### PR TITLE
Add password confirmation constraints

### PR DESCRIPTION
**Pull request by Nainterceptor**
Checklist

* [x] I've added a mention in CHANGELOG.md

### CODE REVIEW

File: components/form/password-confirmation.vue [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+      name="password_confirmation"
+      data-vv-as="Mot de passe (confirmation)"
+      :placeholder="'Tapez votre mot de passe (confirmation)'"
+      @input="input"
+    >
+    <p
+      v-if="!isValid"
+      class="help is-danger"
+    >
+      La confirmation de mot de passe ne correspond pas.
```
Comment (Scttpr) [inline components/form/password-confirmation.vue]: Wording : "Les deux mots de passe ne correspondent pas" || "La confirmation du mot de passe ne corresppond pas"

File: pages/edit-account.vue [hunk 2]
```diff
@@ -366,6 +363,10 @@ export default {
   },
   methods: {
     async sendForm(sendToken = false) {
+      if ((this.fields.password || '') !== (this.fields.password_confirm || '')) {
+        this.$toast.error('Le mot de passe et sa confirmation ne correspondent pas.');
+        return;
```
Comment (Scttpr) [inline pages/edit-account.vue]: Could empty inputs here ?

Comment (Nainterceptor) [reply pages/edit-account.vue]: Not expected, if we clean, we cannot compare visually lengths of passwords

File: components/form/password-confirmation.vue (hunk 1 above)
Comment (Nainterceptor) [reply components/form/password-confirmation.vue]: Fixed

File: components/form/password-confirmation.vue [hunk 3]
```diff
@@ -0,0 +1,58 @@
+<template>
+  <div>
+    <input
+      v-validate="'required'"
+      v-bind="$attrs"
```
[the remark this answers is not in the document]
Comment (Nainterceptor) [reply components/form/password-confirmation.vue]: Done !

**Pull request merged**